toy-project/recoil/atoms.ts

```diff
@@ -0,0 +1,46 @@
+import { ProfileType, followList } from '@/types/types';
+import { atom } from 'recoil';
+import { recoilPersist } from 'recoil-persist';
+const { persistAtom } = recoilPersist();
+
+export const profileInfo = atom<ProfileType>({
+  key: 'profileInfo',
+  default: {
+    login: '',
+    avatar_url: '',
+    bio: '',
+    followers: 0,
+    following: 0,
+  },
+});
+
+export const followerList = atom<followList[]>({
+  key: 'followerList',
+  default: [{ id: 0, login: '', avatar_url: '' }],
+});
+
+export const followingList = atom<followList[]>({
+  key: 'followingList',
+  default: [{ id: 0, login: '', avatar_url: '' }],
+});
+
+export const followState = atom<string>({
+  key: 'followState',
+  default: 'follow',
+});
```

Comment:
컴포넌트의 사용처에서 확인해보니 `'follow' | 'not'` 두 가지 경우로 사용되는 것 같은데요, 그렇다면 atom의 타입을 string으로 적어주기 보다는 아래처럼 나타내는게 더 좋을 것 같아요.(string literal type, enum 타입 이라고도 불러요) 실제 IDE에서 타입 추론을 해주어 휴먼에러를 방지해주기 위함이에요.

그리고 follow, not 의 두 가지 논리만 존재한다면 `boolean` 타입으로 두어도 괜찮을 것 같습니다! 

```suggestion
export const followState = atom<'follow' | 'not'>({
  key: 'followState',
  default: 'follow',
});
```